Update endpoint-as2-receive.adoc (#79)
mulesoft/docs-partner-manager: modules/ROOT/pages/endpoint-as2-receive.adoc

|Describe any important details for this endpoint. +
Required: No +

| AS2 URL
| The HTTP or HTTPS address where you receive AS2 messages.

> When deploying to CloudHub with a public Shared Load Balancer, the URL is automatically generated when you deploy a message flow that uses this endpoint.

Reviewer: unclear reference: the URL is not deploying....
```suggestion
The URL is automatically generated when you use a shared load balancer to deploy a message flow to CloudHub.
```

Author: This text has been removed as it is no longer accurate.

Reviewer 2: We do not want to bring the topic of load balancer at all. 
We just need to have the below with respect to CloudHub.

`When deploying to CloudHub the URL is automatically generated when you deploy a message flow that uses this endpoint. You can see it by selecting the receiving endpoint from your deployed message flow.`

> To get the external facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub with Dedicated load balancer) refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer

Reviewer: ```suggestion
To get the external-facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub by using a dedicated load balancer), refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer.
```
...but do we really want to introduce a goal and then send the audience off to some other source to tell how to achieve that goal?

Author: It's a set of steps only related to load balancer so yes, I think so.

> To get the external-facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub with Dedicated load balancer) refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer.

Reviewer 2: This sentence needs to be removed.

> You view it by selecting the receiving endpoint from your deployed message flow.

Reviewer: View what?

Author: Fixed.

> If deploying to On-premises Mule with a Firewall and Load Balancer, work with your hosting or Network Security team to obtain the externally accessible DNS /host name where the Mule runtime server is executing, and replace {runtime_host} in the URL. For example, if the DNS host name is b2b.mycompany.com - the AS2 Receiver URL is: https://b2b.mycompany.com/receive-as2/

Reviewer: ```suggestion
If deploying to on-premises Mule with a firewall and load balancer, work with your hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule runtime server is executing, and replace {runtime_host} in the URL. For example, if the DNS host name is `b2b.mycompany.com`, the AS2 receiver URL is `https://b2b.mycompany.com/receive-as2/`.
```

Author: Fixed.

| <Host> public certificate
| This is your partner’s digital certificate (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
NOTE: If a single certificate is uploaded, Partner Manager prevents you from deleting your only certificate. +

Required: Yes +

| <Host> private key file
| This is your private key.

Required: Yes +

| Key passphrase

> |  This is the passphrase for your private key

Reviewer: ```suggestion
| This is the passphrase for your private key.
```

Author: Fixed.

Required: Yes +

= Supported Signing and Encryption Algorithms

> AS2 receive endpoints deployed from Partner Manager support the below signing & encryption algorithms.

Reviewer: ```suggestion
AS2 receive endpoints deployed from Partner Manager support the following signing and encryption algorithms.
```

Author: REmoved.
